server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/BasicAuthenticationTokenManager.scala

```diff
@@ -0,0 +1,40 @@
+package org.apache.james.jmap.http
+
+import java.util.Base64
+
+import javax.inject.Inject
+import org.apache.james.core.Username
+import org.apache.james.jmap.http.BasicAuthenticationTokenManager.PREFIX
+import org.apache.james.user.api.UsersRepository
+
+object BasicAuthenticationTokenManager {
+  val PREFIX: String = "Basic "
+}
+
+class BasicAuthenticationTokenManager @Inject() (val usersRepository: UsersRepository) {
```

Comment:
IMO we are not enough limiting the visibility of methos here. We should rely on `private` things as much as possible.